modules/tinymce/src/core/main/ts/api/dom/Selection.ts

```diff
@@ -257,6 +257,12 @@ const EditorSelection = (dom: DOMUtils, win: Window, serializer: DomSerializer,
     return !sel || rng.collapsed;
   };
 
+  /**
+   * Checks if the current selection start and end containers are editable within their parent contexts.
+   *
+   * @method isEditable
+   * @return {Boolean} Will be true if the selection is editable and false if it's not.
```

Comment:
Just to be consistent with the `DOMUtils` doc

```suggestion
   * @return {Boolean} Will be true if the selection is editable and false if it's not editable.
```